examples/message-ubench/main.pony

```diff
@@ -151,6 +177,10 @@ actor SyncLeader
     _partial_count = 0
     _waiting_for = _ps.size()
 
+    if _run_done then
+      _timers.cancel(_interval_timer)
```

Comment:
this isn't needed. the timer isn't reset after its first used in `RunTimerNotify.apply`

Reply:
This is canceling the Tick not RunTimerNotify.